pifpaf/drivers/mysql.py

```diff
@@ -23,11 +26,24 @@ def _setUp(self):
         tempdir = os.path.join(self.tempdir, "tmp")
         os.mkdir(datadir)
         os.mkdir(tempdir)
+
+        c, out = self._exec(["whoami"], stdout=True, ignore_failure=True,
```

Comment:
I think you could use `os.getlogin()` here instead of relying on an external program.